ci-operator/step-registry/osd/create/create/osd-create-create-ref.yaml

```diff
@@ -31,6 +32,9 @@ ref:
   - name: CLUSTER_MULTI_AZ
     default: "false"
     documentation: Set to 'true' if you want to deploy a cluster across multiple availability zones.
+  - name: CLUSTER_TIMEOUT
+    default: "5400"
```

Comment:
Just a question/observation. The `timeout` in line 10 is set to `2h0m0s`. The `CLUSTER_TIMEOUT` environment variable's `default` is set to `5400`. I assume that's seconds. Which is 1.5 hours.

How are these properties related?

Reply:
They are not related. The ```timeout``` in line 10 is a timeout for the osd cluster create step.
The ```CLUSTER_TIMEOUT``` variable is a timeout for the cluster to obtain the status ```ready```. If the cluster is not ```ready``` in 5400 seconds (1.5 hours) the execution stops.

Reply:
> > @bradmwilliams @jupierce could you ptal? Thanks!
> 
> While I think these changes look OK, I'm not quite sure I quite understand the need for it. In my experience, the success of getting a cluster up is more important that shaving a few minutes out of the process.
> 
> If OSD is anything like the other cloud platforms, where the installation time is quite variable, time boxing it seems kind of risky.

@bradmwilliams 
I totally agree with your statement of timeboxing being risky approach. However, Within CVP MSP effort we would not want to wait for more cluster creation for beyond certain time. We would rather create a new cluster. 
If adding default values is a concern, we can always set to null value and based on the parameter passed to the step we can customize script to use the CLUSTER_TIMEOUT or not. 
If CLUSTER_TIMEOUT is null we would not timeout at all. 
Would that work ?

Reply:
Right now, the step timeout is set to: `timeout: 2h0m0s` (which is the default value for steps anyway).  I'd recommend making the default `CLUSTER_TIMEOUT` value `7200`.  This way you are not changing any existing behavior.  Then, in your specific jobs, you can tweak the `CLUSTER_TIMEOUT` values to your liking.

Reply:
@bradmwilliams we can set it to 7200

Reply:
it is set to 7200, @bradmwilliams could you ptal?